Gemfile

```diff
@@ -2,3 +2,5 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in astral_plane.gemspec
 gemspec
+
+gem 'pry', '~> 0.12.2'
```

Comment:
Most gem developers put this in the gemspec file and not the gemfile. They will also use `spec.add_development_dependency` to keep it out of your production build.